logparser/logparser.go

```diff
@@ -62,7 +64,14 @@ func (p *logParser) parse() (*LogEntry, error) {
 		return nil, fmt.Errorf("failed to skip to MSG: %s", err)
 	}
 
-	message := app + "[" + process + "]: " + string(p.b[p.cursor:])
+	var msg string
+	if process == "router" {
+		msg = parseRouterLogMessage(string(p.b[p.cursor:]))
```

Comment:
Makes me nervous that we have to special-case different log types. This means we need to write code for dyno metrics, postgres metrics, redis metrics, etc.